code/reports/rmdchunks/plot-forecasts.Rmd

```diff
@@ -36,12 +36,11 @@ for (forecast_date in as.character(forecast_dates)) {
         facet_formula =  rlang::expr(~ !!facetting_var),
         ncol = 3,
         allow_truth_without_pred = FALSE,
-        scales = "free") + 
+        scales = "free_y") + 
         ggplot2::theme(legend.position = "bottom", 
                        strip.placement = "outside") + 
         scale_y_continuous(labels = scales::comma) + 
-        expand_limits(y = 0) + 
-        coord_cartesian(ylim = c(0, NA))
```

Comment:
I removed this because it doesn't seem to serve any purpose but I might be wrong. Do you have any idea why it might be necessary?

Reply:
Yes, we sometimes end up with negative values because the incident case/death numbers are based on diffs of cumulative numbers, and sometimes these go down if e.g. duplicates are removed etc. In that case I think we want to not show the negative numbers.